docs/reference/api/docker_remote_api.md

```diff
@@ -116,6 +116,8 @@ This section lists each version from latest to oldest.  Each listing includes a
 
 [Docker Remote API v1.25](docker_remote_api_v1.25.md) documentation
 
+* `POST /containers/create` now takes `AutoRemove` in HostConfig, auto-removal will be done on daemon side.
```

Comment:
How about this? Maybe it's too wordy?

> * `POST /containers/create` now takes `AutoRemove` in HostConfig, to enable auto-removal
  of the container on daemon side when the container's process exits.